packages/base-driver/lib/express/middleware.js

```diff
@@ -76,14 +76,15 @@ export function fixPythonContentType(basePath) {
  * @returns {any}
  */
 export function handleLogContext(req, res, next) {
+  /** @type {import('node:async_hooks').AsyncLocalStorage<Record<string, any>>|undefined} */
   const contextStorage = global._global_npmlog?.asyncStorage;
   if (!contextStorage) {
     return next();
   }
 
   const sessionIdMatch = SESSION_ID_PATTERN.exec(req.url);
   contextStorage.enterWith(
-    sessionIdMatch ? {sessionSignature: sessionIdMatch[1].substring(0, 8)} : {}
+    sessionIdMatch ? {sessionSignature: sessionIdMatch[1]} : {}
```

Comment:
I would rather keep this one and add one more context variable elsewhere

Reply:
The sessionSignature field has been restored to 8 characters.
and added "sessionId" field to save full id.

Reply:
yes, I like this way it has much more flexibility 👍